### PR TITLE
Applied Changes as Required for Bevy 0.12

### PR DESCRIPTION
**Pull request by bushrat011899**
# Objective

Bare minimum changes required to move to Bevy 0.12 from 0.11.

# Solution

- Bumped Bevy to version 0.12
- Split `ComputedVisibility` into `ViewVisibility` and `InheritedVisibility`
- Updated `add_schedule` calls to use new convention
- Replaced `EntityMap` with `HashMap`

### CODE REVIEW

**Comment by johanhelsing** [comment]
Once this and #80 is in, we should probably do another crates release? Eager to update tutorials and other crates to all the great new stuff!

**Comment by gschup** [comment]
That sounds great! Will go over the PRs over this week

File: src/lib.rs [hunk 1]
```diff
@@ -176,20 +176,19 @@ impl<C: Config> Default for GgrsPlugin<C> {
...
-
         app.init_resource::<RollbackFrameCount>()
             .init_resource::<ConfirmedFrameCount>()
             .init_resource::<MaxPredictionWindow>()
             .init_resource::<RollbackOrdered>()
             .init_resource::<LocalPlayers>()
             .init_resource::<FixedTimestepData>()
-            .add_schedule(GgrsSchedule, schedule)
-            .add_schedule(ReadInputs, Schedule::new())
+            .edit_schedule(GgrsSchedule, |schedule| {
```
Comment (gschup) [inline src/lib.rs:185]: How is this change related to updating to bevy 0.12? I see why it is nice to have ambiguity being an error, but that is an issue separate from updating to 0.12, if I understand correctly.

**Review by gschup — APPROVED** [review]
Thanks for updating! Seems we got lucky this time around and the changes are minor. Just one small comment, otherwise this looks great!

**Review by johanhelsing — APPROVED** [review]
Looks great to me

File: src/lib.rs (hunk 1 above)
Comment (johanhelsing) [reply src/lib.rs:185]: I think it was just done further up in the file before?

Comment (gschup) [reply src/lib.rs:185]: Oh, I see! silly me :)
